movies/src/test/java/movies/CustomerTest.java

```diff
@@ -0,0 +1,44 @@
+package movies;
+
+import static org.junit.Assert.*;
```

Comment:
Import statements are divided into the following groups, in this order, with each group separated by a single blank line:
1. java imports
2. javax imports
3. Third-party imports, one group per top-level package, in ASCII sort order
for example: android, com, junit, org, sun
4. local packages imports.
5. All static imports in a single group

Example:
```java
 import java.util.List;
 import java.util.ArrayList;
 import java.util.Date;

 import javax.swing.JFrame;
 import javax.swing.JPanel;
 import javax.swing.event.TableModelListener;

 import org.apache.commons.httpclient.cookie.CookiePolicy;
 import org.apache.commons.httpclient.cookie.CookieSpec;

 import your.own.packagename.here.SomeClass;
 import your.own.packagename.here.OtherClass;

 import static org.junit.Assert.assertEquals;
```